tracer/src/Datadog.Trace/AspNet/SharedItems.cs

```diff
@@ -17,9 +17,9 @@ internal static class SharedItems
         private static readonly Func<Stack<Scope>, Scope> Pop = stack => stack.Pop();
         private static readonly Func<Stack<Scope>, Scope> Peek = stack => stack.Peek();
 
-        internal static void PushScope(HttpContext context, string key, Scope item)
+        internal static void PushScope(HttpContext? context, string key, Scope item)
```

Comment:
I wonder whether we _should_ make this one nullable 🤔 On the one hand, it's "safe", so we can, but it also _implies_ that it "still works" if it's null to callers, so I don't know if we want that or not 🤔

Reply:
I put in back to non nullable in dcd9e9d26b231b02473cd3b1928cfac81bdcd642 to not incent anyone to give a null HttpContext 🤣